### PR TITLE
Add label validation to prs

### PR DESCRIPTION
**Pull request by nhorman**
Add a check to ensure that we don't merge prs that don't have severity: urgent or approval: ready to merge set

### CODE REVIEW

File: review-tools/ghmerge [hunk 1]
```diff
@@ -157,24 +158,48 @@ from __future__ import print_function
...
+        rtm_set=1
+    if (l["name"] == "severity: urgent"):
+        urgent_set=1
+if (rtm_set == 0 and urgent_set == 0):
+    print("Not ready to merge")
+' <$PR_URL_CONTENTS)
+
+if [ "$REPO_CHECK" == "Not ready to merge" ]
+then
+    >&2 echo "This pr has neither the urgen or ready to merge flag set, Won't merge"
```
Comment (t8m) [inline review-tools/ghmerge:191]: Typo: urgent

**Review by t8m — APPROVED** [review]

**Comment by levitte** [comment]
Side note: all those python scriplets to filter json output could easily be refactored into `jq` commands.  That would be a worthy modernization...

**Review by levitte — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you.

**Comment by nhorman** [comment]
@levitte I had actually considered doing that, but wasn't sure how widely distributed jq was on our various development systems.  if there is general consensus on doing that, I can follow up with a refactor

**Comment by levitte** [comment]
I would imagine that jq is a commonly available on most Linuxes at this point.  It's been a commonly available on Debian since [buster](https://packages.debian.org/buster/jq), which goes like what, 5 years back now?  It's also been commonly available on Ubuntu since [20.04LTS](https://packages.ubuntu.com/search?keywords=jq).

**Comment by levitte** [comment]
... and the availability doesn't stop there: https://jqlang.github.io/jq/download/

**Comment by nhorman** [comment]
yeah, availability isn't a problem, was more concerned with the fact that its not part of the default install.  But that can be fixed with an additional check.  I'll make the change

**Comment by levitte** [comment]
Additional check sounds good

**Pull request closed without merging**